include/backup_restore.hpp

```diff
@@ -25,10 +25,10 @@ class BackupRestore
     /**
      * @brief Constructor.
      *
-     * @param[in] i_parsedSystemJson - Parsed system config JSON.
+     * @param[in] i_sysCfgJsonObj - Parsed system config JSON.
```

Comment:
Constructor is throwing exception. That needs to be updated now.

Reply:
done.

Reply:
The correct wat is to add "@throw" which is a valid doxygen tag.
Please use that.

Reply:
Updated